src/style.css

```diff
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.searchBar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.searchBarBig {
+  width: 100%;
+  height: 70px;
+}
+
+.searchBarSmall {
+  width: 100%;
+  height: 42px;
+}
+
+.label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.labelBig:after {
+  background-image: url("images/Search.svg");
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.labelSmall:after {
+  background-image: url("images/Search.svg");
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  background-repeat: no-repeat;
+  background-size: contain;
+
+}
+
+.input {
+  width: 100%;
```

Comment:
it is a default style for clock elements, you can add `display: block;` and remove it on labels